roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -61,33 +62,69 @@ class Interface(object):
             return True
 
         # veth interfaces named tap... belong to Neutron
-        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname) and \
-                self.iftype() == 'veth':
-            return True
+        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname) and self.iftype() == 'veth':
+               return True
```

Comment:
`roles/neutron-common/templates/usr/local/bin/neutron-reset:66:16: E111 indentation is not a multiple of four`